examples/pytorch/01-Getting-started.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    model.initialize(train_loader)
</pre><p>can we add some explanation why we do need <code>model.initialize()</code> step? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>